packages/yoga/src/Input/web/Field.jsx

```diff
@@ -88,6 +88,8 @@ const Field = styled.input`
     &:disabled {
       cursor: not-allowed;
       color: ${colors.text.disabled};
+      -webkit-text-fill-color: ${colors.text.disabled};
```

Comment:
As a discovery, could you check if in case you add just `text-fill-color`, styled-component is going to to handle this `-webkit` and other types of fallback.

Letting styled-component handle that will cover other typer of browser engines, not only safari/chrome ones

Reply:
![image](https://user-images.githubusercontent.com/14081572/175695760-85e0026e-6b8b-46df-8fad-3867173c6942.png)


good catch, it worked fine